tools/CMakeLists.txt

```diff
@@ -229,20 +229,21 @@ if(IREE_BUILD_COMPILER)
     )
   endif()
 
+  # Tablegen binaries are special snowflakes among special snowflakes.
```

Comment:
> Fixes the iree-tblgen target to be correct in shared-library and installed builds.

I wonder if https://github.com/openxla/iree/issues/5420 is fixed now

Reply:
Probably not. Different weird thing ...